src/parser/clauses/return.c

```diff
@@ -12,31 +12,42 @@ AST_ReturnElementNode* New_AST_ReturnElementNode(AST_ArithmeticExpressionNode *e
 	AST_ReturnElementNode *returnElementNode = (AST_ReturnElementNode*)malloc(sizeof(AST_ReturnElementNode));
 	returnElementNode->exp = exp;
 	returnElementNode->alias = NULL;
+    returnElementNode->asterisks = 0;
 
 	if(alias != NULL) returnElementNode->alias = strdup(alias);
 	
 	return returnElementNode;
 }
 
+AST_ReturnElementNode* New_AST_ReturnElementExpandALL() {
+    AST_ReturnElementNode *returnElementNode = New_AST_ReturnElementNode(NULL, NULL);
+    returnElementNode->asterisks = 1;
+	return returnElementNode;
+}
+
 AST_ReturnNode* New_AST_ReturnNode(Vector *returnElements, int distinct) {
 	AST_ReturnNode *returnNode = (AST_ReturnNode*)malloc(sizeof(AST_ReturnNode));
 	returnNode->returnElements = returnElements;
 	returnNode->distinct = distinct;
-	return returnNode;
+    return returnNode;
```

Comment:
I hate what it will do to our Git history, but we really need to switch to tabs and revise the entire repo accordingly.